app/Actions/Docker/GetContainersStatus.php

```
@@ -98,11 +98,13 @@ public function handle(Server $server, ?Collection $containers = null, ?Collecti
             $containerStatus = data_get($container, 'State.Status');
-            $containerHealth = data_get($container, 'State.Health.Status', 'unhealthy');
+            $containerHealth = data_get($container, 'State.Health.Status');
             if ($containerStatus === 'restarting') {
-                $containerStatus = "restarting ($containerHealth)";
+                $healthSuffix = $containerHealth ?? 'unknown';
+                $containerStatus = "restarting ($healthSuffix)";
             } else {
-                $containerStatus = "$containerStatus ($containerHealth)";
+                $healthSuffix = $containerHealth ?? 'unknown';
+                $containerStatus = "$containerStatus ($healthSuffix)";
             }
```

PHPMD (2.15.0)

The method handle uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)